### PR TITLE
[Security Content] Add Windows Investigation Guides

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

This PR adds Investigation Guides to the following rules:

* First Time Seen Commonly Abused Remote Access Tool Execution
* FirstTime Seen Account Performing DCSync
* PowerShell Invoke-NinjaCopy script
* PowerShell Script with Encryption/Decryption Capabilities
* Process Termination followed by Deletion
* Potential Secure File Deletion via SDelete Utility
* Renamed Utility Executed with Short Program Name (Previously: Suspicious Execution - Short Program Name)
* Suspicious Zoom Child Process
* Enumerating Domain Trusts via DSQUERY.EXE
* Enumerating Domain Trusts via NLTEST.EXE

### CODE REVIEW

File: rules/windows/command_and_control_new_terms_commonly_abused_rat_execution.toml
Comment (joepeeples) [inline rules/windows/command_and_control_new_terms_commonly_abused_rat_execution.toml:39]: Nit: This isn't exactly a step or action the reader can take; it just describes a possible scenario. Maybe:
```suggestion
  - If the tool is not approved for use in the organization, the employee could have been tricked into installing it and providing access to a malicious third party. Investigate whether this third party could be attempting to scam the end-user or gain access to the environment through social engineering.
```

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml
Comment (joepeeples) [inline rules/windows/credential_access_dcsync_newterm_subjectuser.toml:29]:
```suggestion
Adversaries can use the DCSync technique that uses Windows Domain Controller's API to simulate the replication process from a remote domain controller, compromising major credential material such as the Kerberos krbtgt keys that are used legitimately for creating tickets, but also for forging tickets by attackers. This attack requires some extended privileges to succeed (DS-Replication-Get-Changes and DS-Replication-Get-Changes-All), which are granted by default to members of the Administrators, Domain Admins, Enterprise Admins, and Domain Controllers groups. Privileged accounts can be abused to grant controlled objects the right to DCsync/Replicate.
```

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml
Comment (joepeeples) [inline rules/windows/credential_access_dcsync_newterm_subjectuser.toml:45]:
```suggestion
- Administrators may use custom accounts on Azure AD Connect; investigate if this is part of a new Azure AD account setup, and ensure it is properly secured. If the activity was expected and there is no other suspicious activity involving the host or user, the analyst can dismiss the alert.
```

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml
Comment (joepeeples) [inline rules/windows/credential_access_dcsync_newterm_subjectuser.toml:52]: (Verb is singular if the compound subject uses `or`.)
```suggestion
- If the entire domain or the `krbtgt` user was compromised:
```

File: rules/windows/credential_access_dcsync_replication_rights.toml
Comment (joepeeples) [inline rules/windows/credential_access_dcsync_replication_rights.toml:52]:
```suggestion
- If the entire domain or the `krbtgt` user was compromised:
```

File: rules/windows/defense_evasion_posh_encryption.toml
Comment (joepeeples) [inline rules/windows/defense_evasion_posh_encryption.toml:27]: The "from ... to" format gets a little confusing with a long phrase, maybe try something like:
```suggestion
PowerShell offers encryption and decryption functionalities that can be abused by attackers for a variety of purposes, such as concealing payloads and C2 communications to bypass security protections and performing ransomware capabilities.
```

File: rules/windows/defense_evasion_sdelete_like_filename_rename.toml
Comment (joepeeples) [inline rules/windows/defense_evasion_sdelete_like_filename_rename.toml:24]:
```suggestion
SDelete is a tool primarily used for securely deleting data from storage devices, making it unrecoverable. Microsoft develops it as part of the Sysinternals Suite. Although commonly used to delete data securely, attackers can abuse it to delete forensic indicators and remove files as a post-action to a destructive action such as ransomware or data theft to hinder recovery efforts.
```

File: rules/windows/defense_evasion_suspicious_zoom_child_process.toml
Comment (joepeeples) [inline rules/windows/defense_evasion_suspicious_zoom_child_process.toml:50]:
```suggestion
By examining the specific traits of Windows binaries -- such as process trees, command lines, network connections, registry modifications, and so on -- it's possible to establish a baseline of normal activity. Deviations from this baseline can indicate malicious activity, such as masquerading, and deserve further investigation.
```

**Review by joepeeples — COMMENTED** [review]
Just a few small suggested edits -- thanks for preparing this content!

FYI, to streamline reviews, the Security Docs team will now have a single tech writer reviewing most PRs for prebuilt rules & investigation guides. No need to wait for reviews from the other writers on this one, thanks!

**Review by imays11 — APPROVED** [review]
Great work

**Review by joepeeples — APPROVED** [review]
LGTM! Thanks again for preparing this content!

**Review by Aegrah — APPROVED** [review]
Very nice work, looks good!

**Pull request merged**
